<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>Resume - Natalia Faraj Murad</title>
        <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v6.3.0/js/all.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
        <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
    </head>
    <body id="page-top">
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="#page-top">
                <span class="d-block d-lg-none">Natalia Faraj Murad</span>
                <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="assets/img/profile.jpg" alt="..." /></span>
            </a>
            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarResponsive">
                <ul class="navbar-nav">
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about">About</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#experience">Experience</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#education">Education</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#skills">Skills</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#interests">Interests</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#publications">Publications</a></li>
                </ul>
            </div>
        </nav>
        <!-- Page Content-->
        <div class="container-fluid p-0">
            <!-- About-->
            <section class="resume-section" id="about">
                <div class="resume-section-content">
                    <h1 class="mb-0">
                        Natalia
                        <span class="text-primary">Faraj Murad</span>
                    </h1>
                    <div class="subheading mb-5">
                        San Francisco · CA ·
                        <a href="mailto:name@email.com">nataliafmurad@gmail.com</a>
                    </div>
                    <style>
                        .paragraph {
                            text-align: justify;
                            margin-bottom: 20px;
                            text-indent: 20px;
                        }
                    </style>
                    <p class="paragraph">
                        I am a highly skilled bioinformatician and data scientist with a strong background in genetics and molecular biology.
                        My professional journey includes working as a Postdoctoral Researcher at the Buck Institute for Research on Aging, where I focused on bioinformatics analysis of various types of data, including metabolomics, proteomics, metagenomics, scRNA-seq, bulk RNA-seq, and spatial transcriptomics.
                        I excelled in identifying senescent biomarkers, brain disease association factors, and aging signatures.
                    </p>
                    <p class="paragraph">
                        My expertise also extends to statistical analysis, experimental design, and the development of machine learning models and automated pipelines for large-scale data analysis.   
                        Throughout my academic journey, I received prestigious research grants and has published numerous articles in esteemed scientific journals, including papers on probabilistic graphical models applied to biological networks and the mechanisms behind polyphagia in pest insects.<br>
                    </p>
                    <p class="paragraph">  
                            In addition to my research expertise, I have continuously pursued professional development, completing various courses in data science, genomic data analysis, Python, big data, and more.
                            I possess advanced programming skills in R, Python, SQL, and Bash, and is proficient in tools such as Docker, Snakemake, Bioconductor, and machine learning libraries.                   
                            With a strong foundation in bioinformatics, data analysis, and research methodologies, I am well-equipped to contribute to cutting-edge scientific projects and make significant advancements in the field of biological research.
                    </p>
                    <div class="social-icons">
                        <a class="social-icon" href="https://www.linkedin.com/in/natmurad/"><i class="fab fa-linkedin-in"></i></a>
                        <a class="social-icon" href="https://github.com/natmurad"><i class="fab fa-github"></i></a>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Experience-->
            <section class="resume-section" id="experience">
                <div class="resume-section-content">
                    <h2 class="mb-5">Experience</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Data Scientist & Bioinformatician</h3>
                            <div class="subheading mb-3">Buck Institute for Research on Aging</div>
                            <p>Bring to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution. User generated content in real-time will have multiple touchpoints for offshoring.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">January 2022 - Present</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Data Scientist</h3>
                            <div class="subheading mb-3">University of Campinas</div>
                            <p>Data management, statistical analysis, and bioinformatics. My work involved identifying genes related to biomass yield and bioethanol production, as well as exploring the influence of microorganisms on ethanol production through metagenomics analysis. I also colaborated writing grants and administrative support.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">September 2018 - October 2021</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                </div>
            </section>
            <hr class="m-0" />
            <!-- Education-->
            <section class="resume-section" id="education">
                <div class="resume-section-content">
                    <h2 class="mb-5">Education</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">University of Campinas</h3>
                            <div class="subheading mb-3">PhD in Genetics & Molecular Biology - Bioinformatics</div>
                            <div>Thesis: Gene regulatory networks to identify adaptations of two <i>Spodoptera frugiperda</i> strains to different diets </div>
                            <p>Activities: I focused on gene regulatory networks and multiomics analysis. I developed innovative methods to improve the identification of responses to different diets and conditions in <i>S. frugiperda</i> (a moth). My research involved integrating biological data at various levels, utilizing SQL databases, and applying clustering methods and enrichment analyses.<br></p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">March 2013 - March 2018</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">University of Campinas</h3>
                            <div class="subheading mb-3">MSc in Genetics & Molecular Biology - Bioinformatics</div>
                            <div>Dissertation: Gene regulatory networks of the sucrose metabolism in sugarcane using bayesian networks</div>
                            <p>Activities: I identified the genes closely interacting with the sucrose phosphate synthase which helps to develop strategies to improve sugarcane for sugar and alcohol production. I mapped how the genes related to sugar production were changing their expression in response to cold as well as which pathways were being affected.<br></p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">March 2011 - March 2013</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Federal University of Lavras</h3>
                            <div class="subheading mb-3">Bachelor of Biological Sciences</div>
                            <div>Dissertation: Mixed models applied to microarray data in humans</div>
                            <p>Activities: I applied different approaches of Mixed Linear Models to microarray data and evaluated which strategy was working better. It allowed to develop my own pipeline for microarray data analysis as well as model fitting evaluation.<br></p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">August 2002 - May 2006</span></div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Skills-->
            <section class="resume-section" id="skills">
                <div class="resume-section-content">
                    <h2 class="mb-5">Skills</h2>
                    <div class="subheading mb-3">Programming Languages & Tools</div>
                    <ul class="list-inline dev-icons">
                        <li class="list-inline-item"><i class="fa-brands fa-python"></i></li>
                        <li class="list-inline-item"><i class="fa-solid fa-registered"></i></li>
                        <li class="list-inline-item"><i class="fa-brands fa-github"></i></li>
                        <li class="list-inline-item"><i class="fa-brands fa-linux"></i></li>
                        <li class="list-inline-item"><i class="fa-duotone fa-database"></i></li>
                    </ul>
                    <div class="subheading mb-3">Workflow</div>
                    <ul class="fa-ul mb-0">
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Data analysis: RNA-seq, Single-cell, ATAC-seq, Spatial transcriptomics, Proteomics, Metabolomics, Metagenomics (shotgun & 16S)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Multi-omics, Data integration, Systems Biology
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Machine Learning & Predictive Models: Supervised, Unsupervised and Reinforcement methods. Linear models, k-neighbors, k-means, SVM, Decision trees, Random forest, Boosting (XgBoost, LightGBM) etc.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Data Management & Data Manipulation: pandas, numpy, scipy, dplyr, reshape, data.table, tidyR, SQL tools and more.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Automated Pipelines: Snakemake, Singularity, Docker, Shell/Bash/AWK scripts, R/Python scripts.
                        </li>
                    </ul>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Interests-->
            <section class="resume-section" id="interests">
                <div class="resume-section-content">
                    <h2 class="mb-5">Interests</h2>
                    <p>Besides being a scientist, I like to be active most of the time. I practice exercises every day. I like going to the gym to do weight training and also practice body combat or boxing. I love outdoor sports like running or cycling. I like being in contact with nature, going for hikes and discovering different places, watching the sunset or the stars.</p>
                    <p class="mb-0">I spend a lot of my free time reading or watching videos about philosophy, psychology and self-improvement. I'm very curious about time, the universe, how everything follows patterns and whether it's possible to predict the future. So I also like to read physics and probability.</p>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Awards-->
            <section class="resume-section" id="publications">
                <div class="resume-section-content">
                    <h2 class="mb-5">Publications</h2>
                    <ul class="fa-ul mb-0" style="text-align: justify;">
                        <li>
                            <span class="fa-li"><i class="fas fa-star text-warning"></i></span>
                        <b>Murad, N. F.</b>; Silva-Brandão, K. L.; Brandão, M. M. Mechanisms behind polyphagia in a pest insect: Responses of <i>Spodoptera frugiperda</i> (J.E. Smith) strains to preferential and alternative larval host plants assessed with gene regulatory networks. BBA Gene Regulatory Mechanisms. 2021. [DOI: 10.1016/j.bbagrm.2021.194687] [PMID: 33561559] 
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-star text-warning"></i></span>
                        <b>Murad, N.F.</b>; Brandão,M.M. Probabilistic Graphical Models Applied to Biological Networks.In: Vischi Winck, F. (eds) Advances in Plant Omics and Systems Biology Approaches. Advances in Experimental Medicine and Biology, vol 1346. Springer, Cham. 2022. [DOI: 10.1007/978-3-030-80352-0_7] 
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-file text-warning"></i></span>
                            Ciamponi, F. E.; Procopio, D. P.; <b>Murad, N. F.</b>; Franco, T. T.; Basso, T. O.; Brandão, M. M. Multi-omics network model reveals key genes associated with p-coumaric acid stress response in an industrial yeast strain. Scientific Reports. 2022 [DOI: 10.1038/s41598-022-26843-2] [PMID: 36577778]
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-file text-warning"></i></span>
                            Almeida, D. A.; Horta, M. A. C.; Ferreira-Filho, J. A.; <b>Murad, N. F.</b>; De Souza, A. P. The synergistic actions of hydrolytic genes reveal the mechanism of <i>Trichoderma harzianum</i> for cellulose degradation. Journal of Biotechnology. [DOI: /10.1016/j.jbiotec.2021.05.001] [PMID: 33992696] 
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-file text-warning"></i></span>
                            Ferreira Filho, J. A., Horta, M. A. C., dos Santos, C. A.; Almeida, D. A.; <b>Murad, N. F.</b>; Mendes, J. S.; Sforça, D. A.; Silva, C. B. C.; Crucello, A.; Souza, A. P. Integrative genomic analysis of the bioprospection of identifying biomarkers regulators and accessory enzymes associated with cellulose degradation in a filamentous fungus (<i>Trichoderma harzianum</i>). BMC Genomics 21, 757, 2020. [DOI: 10.1186/s12864-020-07158-w] [PMID: 33138770] [PMCID: PMC7607812] 
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-file text-warning"></i></span>
                            Silva-Brandão, K. L; <b>Murad, N. F.</b>; Peruchi, A.; Martins, C. H. Z.; Omoto, C.; Figueira, A.; Brandão, M. M.; Trigo, J. R. Transcriptome differential co-expression reveals distinct molecular response of fall- armyworm strains. Pest Management Science. v.76, p.ps.6051, 2020. [DOI:10.1002/ps.6051] [PMID: 32815313] 
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-file text-warning"></i></span>
                            Horta, M. A. C.; Filho, J. A. F.; <b>Murad, N. F.</b>; De Oliveira Santos, E.; Dos Santos, C. A.; Mendes, J. S.; Brandão, M. M.; Azonni, S. F.; De Souza, A. P. Network of proteins, enzymes and genes linked to biomass degradation shared by Trichoderma species. Scientific Reports. v.8, p.1341, 2018. [DOI: 10.1038/s41598-018-19671- w] [PMID: 29358662] [PMCID: PMC5778038] 
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-file text-warning"></i></span>
                            Silva-Brandão, K. L.; Peruchi, A.; Seraphim, N.; Murad, <b>N. F.</b>; Carvalho, R. A.; Farias, J. R.; Omoto, C.; Cônsoli, F. L.; Figueira, A.; Brandão, M. M. Loci under selection and markers associated with host plant and host-related strains shape the genetic structure of Brazilian populations of <i>Spodoptera frugiperda</i> (Lepidoptera, Noctuidae). PLoS One. v. 13, p.e0197378, 2018. [DOI: 10.1371/journal.pone.0197378] [PMID: 29787608] [PMCID: PMC5963752] 
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-file text-warning"></i></span>
                            Bajay, S. K.; Cruz, M. V.; Silva, C. C.; <b>Murad, N. F.</b>; Brandão, M. M.; Souza, A. P. Extremophytes under climate change scenarios: Transcriptional coordination of genes reveals distinct selective responses to equatorial and subtropical environments. Frontiers in Plant Science. v.9, p.1376, 2018. [DOI: 10.3389/fpls.2018.01376] [PMID: 30283484] [PMCID: PMC6156123]
                        </li>
                    </ul>
                </div>
            </section>
        </div>
        <!-- Bootstrap core JS-->
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
